funcx_endpoint/funcx_endpoint/endpoint/endpoint.py

```diff
@@ -397,9 +376,19 @@ def register_endpoint(funcx_client, endpoint_name, endpoint_uuid, endpoint_dir):
     logger.debug("Attempting registration")
     logger.debug(f"Trying with eid : {endpoint_uuid}")
     import funcx_endpoint
-    reg_info = funcx_client.register_endpoint(endpoint_name,
-                                              endpoint_uuid,
-                                              endpoint_version=funcx_endpoint.__version__)
+    if previously_registered:
+        reg_info = funcx_client.register_endpoint(endpoint_name,
+                                                  endpoint_uuid,
+                                                  endpoint_version=funcx_endpoint.__version__)
+    else:
+        try:
+            reg_info = funcx_client.register_endpoint(endpoint_name,
+                                                      endpoint_uuid,
+                                                      endpoint_version=funcx_endpoint.__version__)
+        except Exception:
+            sys.stdout.write('The endpoint cannot be registered, with the given config.\n')
```

Comment:
This probably won't print to screen because we are in a daemon context.

Reply:
This part has been removed because of the removal of the `previously_registered` logic. But FYI, you are right, it will not print to console, but rather to the `interchange.stdout` file, due to the daemon setting.